[0m2021.12.01 22:32:02 INFO  Started: Metals version 0.10.6 in workspace '/Users/RJHallsted/projects/advent-2021/day1' for client Visual Studio Code 1.60.0.[0m
[0m2021.12.01 22:32:02 INFO  time: initialize in 0.69s[0m
[0m2021.12.01 22:32:03 WARN  Build server is not auto-connectable.[0m
[0m2021.12.01 22:32:03 WARN  no build tool detected in workspace '/Users/RJHallsted/projects/advent-2021/day1'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.12.01 22:32:03 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:32:07 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
Dec 01, 2021 10:32:07 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 6
Dec 01, 2021 10:32:07 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 8
Dec 01, 2021 10:32:07 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 13
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 1.69s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 1.69s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 1.95s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 1.97s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 2.03s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 2.12s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 5.45s[0m
[0m2021.12.01 22:32:09 INFO  time: code lens generation in 5.45s[0m
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 64)
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 60)
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 59)
Dec 01, 2021 10:32:34 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.InterruptedException
java.util.concurrent.CompletionException: java.lang.InterruptedException
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:659)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:1962)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:42)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.InterruptedException
	at scala.meta.internal.metals.FutureCancelToken.checkCanceled(FutureCancelToken.scala:29)
	at scala.meta.pc.VirtualFileParams.checkCanceled(VirtualFileParams.java:25)
	at scala.meta.internal.pc.CompletionProvider.$anonfun$completions$1(CompletionProvider.scala:77)
	at scala.collection.Iterator$$anon$10.next(Iterator.scala:461)
	at scala.collection.Iterator.toStream(Iterator.scala:1417)
	at scala.collection.Iterator.toStream$(Iterator.scala:1416)
	at scala.collection.AbstractIterator.toStream(Iterator.scala:1431)
	at scala.collection.Iterator.$anonfun$toStream$1(Iterator.scala:1417)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1173)
	at scala.collection.immutable.Stream$Cons.tail(Stream.scala:1163)
	at scala.collection.immutable.StreamIterator.$anonfun$next$1(Stream.scala:1061)
	at scala.collection.immutable.StreamIterator$LazyCell.v$lzycompute(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator$LazyCell.v(Stream.scala:1050)
	at scala.collection.immutable.StreamIterator.hasNext(Stream.scala:1055)
	at scala.collection.convert.Wrappers$IteratorWrapper.hasNext(Wrappers.scala:32)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:134)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.CollectionTypeAdapter.write(CollectionTypeAdapter.java:40)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.write(TypeAdapterRuntimeTypeWrapper.java:69)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.write(ReflectiveTypeAdapterFactory.java:127)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.write(ReflectiveTypeAdapterFactory.java:245)
	at com.google.gson.Gson.toJson(Gson.java:735)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:423)
	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.write(MessageTypeAdapter.java:55)
	at com.google.gson.Gson.toJson(Gson.java:735)
	at com.google.gson.Gson.toJson(Gson.java:714)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:145)
	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.serialize(MessageJsonHandler.java:140)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:59)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.lambda$handleRequest$1(RemoteEndpoint.java:281)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:656)
	... 9 more

something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 58)
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 57)
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 64)
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 57)
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in List[Int]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 50, 50, 59)
Dec 01, 2021 10:32:52 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 149
[0m2021.12.01 22:33:05 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:33:11 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
Dec 01, 2021 10:34:43 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 453
Dec 01, 2021 10:34:45 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 468
[0m2021.12.01 22:35:56 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:36:00 INFO  time: code lens generation in 1.45s[0m
[0m2021.12.01 22:36:00 INFO  time: code lens generation in 1.47s[0m
[0m2021.12.01 22:36:00 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
Dec 01, 2021 10:36:40 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 927
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala in Option[Int]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/input1.scala, 90, 90, 101)
Dec 01, 2021 10:38:02 PM scala.meta.internal.pc.CompletionProvider expected$1
WARNING: String index out of range: -1
Dec 01, 2021 10:38:27 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1242
[0m2021.12.01 22:39:32 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:40:42 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:40:43 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:40:58 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:41:05 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:41:06 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:41:06 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/input1.scala[0m
[0m2021.12.01 22:41:35 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part1.scala[0m
[0m2021.12.01 22:42:55 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part1.scala[0m
[0m2021.12.01 22:43:11 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 10:43:11 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.util.DuplicateFormatFlagsException: Flags = '0'
java.util.concurrent.CompletionException: java.util.DuplicateFormatFlagsException: Flags = '0'
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:647)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.util.DuplicateFormatFlagsException: Flags = '0'
	at java.util.Formatter$Flags.parse(Formatter.java:4443)
	at java.util.Formatter$FormatSpecifier.flags(Formatter.java:2640)
	at java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2709)
	at java.util.Formatter.parse(Formatter.java:2560)
	at java.util.Formatter.format(Formatter.java:2501)
	at java.util.Formatter.format(Formatter.java:2455)
	at java.lang.String.format(String.java:2940)
	at scala.collection.immutable.StringLike.format(StringLike.scala:354)
	at scala.collection.immutable.StringLike.format$(StringLike.scala:353)
	at scala.collection.immutable.StringOps.format(StringOps.scala:33)
	at scala.meta.internal.metals.formatting.IndentOnPaste$.scala$meta$internal$metals$formatting$IndentOnPaste$$stringRepeat(IndentOnPaste.scala:28)
	at scala.meta.internal.metals.formatting.IndentOnPaste$PastedLine$Plain.reformat(IndentOnPaste.scala:228)
	at scala.meta.internal.metals.formatting.IndentOnPaste$.$anonfun$processLines$2(IndentOnPaste.scala:156)
	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:36)
	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:33)
	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:198)
	at scala.collection.TraversableLike.map(TraversableLike.scala:286)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:279)
	at scala.collection.mutable.ArrayOps$ofRef.map(ArrayOps.scala:198)
	at scala.meta.internal.metals.formatting.IndentOnPaste$.processLines(IndentOnPaste.scala:156)
	at scala.meta.internal.metals.formatting.IndentOnPaste$.contribute(IndentOnPaste.scala:107)
	at scala.meta.internal.metals.formatting.RangeFormattingProvider.$anonfun$format$2(RangeFormattingProvider.scala:65)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionList.loop$1(MetalsEnrichments.scala:224)
	at scala.meta.internal.metals.MetalsEnrichments$XtensionList.acceptFirst(MetalsEnrichments.scala:232)
	at scala.meta.internal.metals.formatting.RangeFormattingProvider.$anonfun$format$1(RangeFormattingProvider.scala:64)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.formatting.RangeFormattingProvider.format(RangeFormattingProvider.scala:50)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$rangeFormatting$1(MetalsLanguageServer.scala:1393)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$apply$2(CancelTokens.scala:26)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

[0m2021.12.01 22:43:14 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 10:43:26 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1760
[0m2021.12.01 22:43:50 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 22:43:51 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 10:44:39 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
Dec 01, 2021 10:45:37 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 2348
[0m2021.12.01 22:47:04 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 10:47:41 PM scala.meta.internal.pc.CompletionProvider expected$1
WARNING: String index out of range: -1
[0m2021.12.01 22:48:36 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 10:49:33 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.MatchCaseCompletions$Parents.<init>(MatchCaseCompletions.scala:412)
	at scala.meta.internal.pc.completions.MatchCaseCompletions$Parents.<init>(MatchCaseCompletions.scala:402)
	at scala.meta.internal.pc.completions.MatchCaseCompletions$CaseKeywordCompletion.<init>(MatchCaseCompletions.scala:68)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:485)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:412)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:30)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:404)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:387)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:30)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:458)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:58)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:125)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:139)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/part2.scala in Array[String]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/part2.scala, 168, 168, 181)
[0m2021.12.01 22:53:20 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 10:53:22 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
[0m2021.12.01 22:53:24 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 22:53:30 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 22:53:51 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/part2.scala in Option[Int]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/part2.scala, 150, 150, 161)
Dec 01, 2021 10:56:39 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
Dec 01, 2021 10:56:40 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
Dec 01, 2021 10:57:27 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object App
java.util.concurrent.CompletionException: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object App
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:659)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:1962)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:42)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object App

something's wrong: no file:///Users/RJHallsted/projects/advent-2021/day1/part2.scala in scala.collection.immutable.Queue[<error>]RangePosition(file:///Users/RJHallsted/projects/advent-2021/day1/part2.scala, 121, 121, 584)
Dec 01, 2021 10:58:02 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object App
java.util.concurrent.CompletionException: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object App
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:659)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:1962)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:42)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object App

Dec 01, 2021 10:59:46 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
Dec 01, 2021 10:59:51 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
[0m2021.12.01 22:59:54 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:00:18 WARN  no build tool detected in workspace '/Users/RJHallsted/projects/advent-2021/day1'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.12.01 23:00:22 INFO  Attempting to connect to the build server...[0m
[0m2021.12.01 23:00:22 INFO  No build server found[0m
[0m2021.12.01 23:00:22 INFO  time: Connected to build server in 39ms[0m
[0m2021.12.01 23:00:22 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:00:22 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part1.scala[0m
[0m2021.12.01 23:00:42 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:00:51 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
Dec 01, 2021 11:01:46 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 5928
[0m2021.12.01 23:01:50 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:02:08 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:03:22 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:03:49 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:04:04 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:04:38 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:04:51 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:04:57 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
[0m2021.12.01 23:05:08 WARN  no build target for: /Users/RJHallsted/projects/advent-2021/day1/part2.scala[0m
